test/views/declaration/ExporterDetailsViewSpec.scala

```diff
@@ -168,23 +131,11 @@ class ExporterDetailsViewSpec extends UnitViewSpec with CommonMessages with Stub
 
       }
 
-      "display error when EORI is provided, but is incorrect" in {
-
-        val view = createView(
-          form(request.declarationType)
-            .fillAndValidate(ExporterDetails(EntityDetails(Some(Eori(TestHelper.createRandomAlphanumericString(18))), None)))
-        )
-
-        view must haveGovukGlobalErrorSummary
-        view must containErrorElementWithTagAndHref("a", "#details_eori")
-        view.getElementById("details_eori-error") must containMessage("declaration.eori.error.format")
-      }
-
       "display error for empty Full name" in {
 
         val view = createView(
           form(request.declarationType)
-            .fillAndValidate(ExporterDetails(EntityDetails(None, Some(Address("", "Test Street", "Leeds", "LS18BN", "England")))))
+            .fillAndValidate(exporter.ExporterDetails(EntityDetails(None, Some(Address("", "Test Street", "Leeds", "LS18BN", "England")))))
```

Comment:
Seen it somewhere already. Could it be trimmed to use class name only?

Reply:
Done